SmurfTestingScripts/smurf_compare_scenarios.py

```diff
@@ -137,7 +137,7 @@ def cleanup():
         exit(1)
 
     workDir = args.scenario.rsplit('/', 1)[0]
-    workDir = os.path.realpath(workDir)
+    workDir = os.path.dirname(os.path.realpath(workDir))
```

Comment:
Shouldn't the argument be args.scenario (without the split)?

Reply:
Waw, I do not know, I can try.

Reply:
It does not work.